src/deck-options.md

```diff
@@ -122,7 +122,7 @@ become overwhelmed by the reviews required.
 Allows you to set an upper limit on the number of reviews to show each day.
 When this limit is reached, Anki will not show any more review cards for the
 day, even if there are some waiting. If you study consistently, this setting can
-help to smooth out occasional peaks in due card counts, and can save you from a
+help to smooth out occasional peaks in due card counts and can save you from a
```

Comment:
This may be personal opinion, but I'm afraid I feel the commas here and below are more readable than the suggested change.